src/algebra/big_operators/multiset.lean

```diff
@@ -58,6 +58,12 @@ lemma prod_cons (a : α) (s) : prod (a ::ₘ s) = a * prod s := foldr_cons _ _ _
 lemma prod_erase [decidable_eq α] (h : a ∈ s) : a * (s.erase a).prod = s.prod :=
 by rw [← s.coe_to_list, coe_erase, coe_prod, coe_prod, list.prod_erase ((s.mem_to_list a).2 h)]
 
+@[simp, to_additive]
+lemma prod_map_erase [decidable_eq ι] {a : ι} (h : a ∈ m) :
```

Comment:
Note that the finset version is called `mul_prod_erase`; can you edit the comment of that lemma to include "see `multiset.prod_map_erase` for the finset version" so that the mismatched naming doesn't impact discoverability?

Reply:
Done.